src/Engine/ProtoCore/RuntimeCore.cs

```diff
@@ -424,5 +436,36 @@ internal int GetCoercionScore(int srcType, int targetType)
         {
             return ClassTable?.ClassNodes[srcType]?.GetCoercionScore(targetType) ?? (int)ProcedureDistance.NotMatchScore;
         }
+
+        internal FunctionGroup GetFuncGroup(string methodName, string className)
+        {
+            int classScope = ClassTable.IndexOf(className);
+            // try to use dynamic classScope
+            /*if (arguments.Count > 0)
```

Comment:
this commented part is supposed to be called at runtime...which we do not do anymore

Reply:
Are you saying this commented code is no longer required to be called at runtime in the new VM? If so, can we remove it or do we need to make sure before we can do so?

Reply:
I was hoping we could use this API only at compile time. 
But it looks like there might be regressions with polymorphism if we skip the commented code

I might have to move it all at runtime again...

Reply:
We can also figure out polymorphism later on (probably will have to move some of the method resolution to runtime).
If this looks good enough for the go/no go testing, I can add a separate jira task to figure out polymorphism

Reply:
Yes, I feel some part of method resolution will always be needed at runtime. For now, we can create a task and defer it.